Sources/PubNub/EventEngine/Core/EventEngine.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  EventEngine.swift
+//
+//  PubNub Real-time Cloud-Hosted Push API and Push Notification Client Frameworks
+//  Copyright © 2019 PubNub Inc.
```

Comment:
wow. A time machine?

Reply:
Indeed :) I copied the header comment instead of relying on autogenerated version